### PR TITLE
PIS-26: create GitHub actions pipeline

### PR DESCRIPTION
**Pull request by kubao503**

### CODE REVIEW

File: .flake8
Comment (kozlowskimaciej) [inline .flake8:1]: This file shouldn't be placed in the top level directory as it's only being used for backend's Python code. And you want to merge @misat1505 's branch, on which he is still working (and has two package files in the top level dir too, which is not acceptable)? Imo you should remove jest-tests job from CI alongside with all of the Michał's commits, and add it back, when the frontend branch gets merged into master.

**Review by kozlowskimaciej — CHANGES_REQUESTED** [review]

File: .flake8
Comment (kubao503) [reply .flake8:1]: Done. Just a couple of force pushes did the trick.

Comment (kubao503) [reply .flake8:1]: Done. Just a couple of force pushes did the trick.

**Comment by kozlowskimaciej** [comment]
Could you try adding jest job back and see if it will pass without any tests? If yes then it can stay

**Comment by kubao503** [comment]
> Could you try adding jest job back and see if it will pass without any tests? If yes then it can stay

No, it cannot pass now

**Review by kozlowskimaciej — APPROVED** [review]

**Pull request merged**